manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0

```

```suggestion
PackageVersion: 4.36.3020.0
```
This is the PackageVersion of the MSI